src/index.html

```diff
@@ -1,22 +1,125 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
```

Comment:
You have duplicate preconnect links for both 'https://fonts.googleapis.com' and 'https://fonts.gstatic.com'. It's unnecessary to have multiple preconnect links for the same domain, as one preconnect per domain is sufficient. Consider removing the redundant links to optimize performance.